test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -175,6 +183,31 @@ public static Set<UserType> extractUDTs(TableMetadata metadata)
         return matches;
     }
 
+    public static void visitUDTs(TableMetadata metadata, Consumer<UserType> fn)
+    {
+        Set<UserType> udts = CassandraGenerators.extractUDTs(metadata);
```

Comment:
I could make this more efficient but figured reusing is best as its been stable...